components/assetcard/index.css

```diff
@@ -13,52 +13,84 @@ governing permissions and limitations under the License.
 /* outer container, unstyled */
 .spectrum-AssetCard {
   /* todo: this isn't quite the size from the XD file as 232px is not a size token, so we use 224px */
-  --spectrum-assetcard-asset-size: var(--spectrum-global-dimension-static-size-2800);
+  --spectrum-assetcard-asset-size: 224px;
```

Comment:
@pfulton Should we go ahead and use 232px like the original design called for since we have to hard-code this value anyway?

Reply:
I did look at Card, with the context that this component will be updated to be more like a quiet card, and card does not have a set width and instead has a min-width of 100px.